app/i18n.ts

```diff
@@ -1,28 +1,32 @@
 import i18n from 'i18next';
+// import i18nextBackend from 'i18next-node-fs-backend';
 import { initReactI18next } from 'react-i18next';
 
-import enUS from '../locales/enUS.json';
-
-const resources = {
-  enUS,
-};
+import en from '../locales/en/translation.json';
+import up from '../locales/up/translation.json'; // this is uppercased and for dev only
 
 i18n
   // pass the i18n instance to react-i18next.
-  .use(initReactI18next)
-  // init i18next
-  .init({
+  // .use(i18nextBackend)
+  .use(initReactI18next);
+
+if (!i18n.isInitialized) {
+  i18n.init({
     debug: true,
-    fallbackLng: 'enUS',
+    fallbackLng: 'en',
```

Comment:
I think long term their may be different en versions. also fine with renaming to `'en-US'`